src/main/target/STM32F7X2/config/ALIENFLIGHTNGF7.config

```diff
@@ -0,0 +1,125 @@
+# Name: ALIENFLIGHTNGF7
+# Description: ALIENFLIGHTNGF7 Standard target configuration
+
+defaults nosave
+
+# Basic I/O
+resource LED 1 C12
+resource LED 2 D02
+resource BEEPER 1 C13
+set beeper_inversion = ON
+set beeper_od = OFF
+
+# Buses
+resource I2C_SCL 1 B06
+resource I2C_SDA 1 B07
+
+resource SPI_SCK 1 A05
+resource SPI_MISO 1 A06
+resource SPI_MOSI 1 A07
+
+resource SPI_SCK 2 B13
+resource SPI_MISO 2 C02
+resource SPI_MOSI 2 C03
+
+resource SPI_SCK 3 B03
+resource SPI_MISO 3 B04
+resource SPI_MOSI 3 B05
+
+# SPI CS pins to pre-initialize
+resource SPI_PREINIT_IPU 1 A04 // MPU6500/ICM20602
```

Comment:
`//` is not a supported comment format for CLI I think

Reply:
@DieHertz Tock this from the NERO.config sample. It works without negative impact but i understand your concern. We can remove it from both configurations.

Reply:
We can use '#' in the middle of a line to add a comment for the line.